include/nccl_ofi_api.h

```diff
@@ -41,6 +41,13 @@ ncclResult_t nccl_net_ofi_iflush_v4(void* recvComm, void* data, int size, void*
 ncclResult_t nccl_net_ofi_closeSend(void *sendComm);
 ncclResult_t nccl_net_ofi_closeRecv(void *recvComm);
 ncclResult_t nccl_net_ofi_closeListen(void *listenComm);
+ncclResult_t nccl_net_ofi_get_mr_key(void* mhandle, uint64_t* mr_key);
```

Comment:
In this commit, I now implement the function, and in case of get_mr_key function pointer equals zero, I return error (as recommended in comment below).